app/controllers/api/v1/hosts_controller.rb

```diff
@@ -40,7 +41,8 @@ def show
         param :operatingsystem_id, String, :desc => "required if host is managed and value is not inherited from host group"
         param :medium_id, String, :desc => "required if not imaged based provisioning and host is managed and value is not inherited from host group"
         param :ptable_id, :number, :desc => "required if host is managed and custom partition has not been defined"
-        param :subnet_id, :number, :desc => "required if host is managed and value is not inherited from host group"
+        param :subnet_id, :number, :desc => "IPv4 subnet, required if host is managed and value is not inherited from host group"
+        param :subnet6_id, :number, :desc => "IPv6 subnet"
```

Comment:
No, you're correct - it is indeed optional, leave it out then. I'm unsure if the subnet_id docs are intended to imply that it's required for (some types of) provisioning, or if it changed since they were written, or they're just wrong.